Tests/scripts/jira_issues.py

```diff
@@ -53,6 +54,14 @@ def jira_file_link(file_name: str) -> str:
     return f"[^{file_name}]"
 
 
+def jira_sanitize_file_name(file_name: str) -> str:
+    return re.sub(r'[^\w-]+', '-', file_name).lower()
```

Comment:
Couldn't it create conflicts with filenames like:
- "my file.txt" (1 space)
- "my  file.txt" (2 spaces)
Both will give my-file.txt
You can remediate by replacing every whitespace with a dash.

Reply:
fixed